### DISCUSSION TITLE
Facing an error

### DISCUSSION THREAD

**Discussion by roshanjoebenny** [NVIDIA-NeMo/NeMo#8802] [Q&A]
When running the below given code
from nemo.collections import nlp as nemo_nlp
I am getting an error as TypeError: unsupported operand type(s) for |: 'type' and 'type'
Please suggest a solution to solve this issue

**Comment by genagurbanguliyev** [comment]
What version of Python and Nemo do you use?

**Reply by roshanjoebenny** [reply]
python 3.10 ,nemo -1.23.0

**Reply by genagurbanguliyev** [reply]
your usage of import seems to be OK. 
I think problem with your installation Nemo.
I usually use python 3.11.8 with Nemo-1.23.0
Firstly when you install Nemo use `conda evn`, and refer to the official link here [https://github.com/NVIDIA/NeMo](https://github.com/NVIDIA/NeMo)
